feat: Add new admin command to print Argo CD initial password (11117) (#11155)
argoproj/argo-cd: docs/user-guide/commands/argocd_admin.md

* [argocd admin dashboard](argocd_admin_dashboard.md)	 - Starts Argo CD Web UI locally
* [argocd admin export](argocd_admin_export.md)	 - Export all Argo CD data to stdout (default) or a file
* [argocd admin import](argocd_admin_import.md)	 - Import Argo CD data from stdin (specify `-') or a file

> * [argocd admin initial-password](argocd_admin_initial-password.md)	 - Prints initial password to login into Argo CD for the first time

Reviewer: ```suggestion
* [argocd admin initial-password](argocd_admin_initial-password.md)	 - Prints initial password to log in to Argo CD for the first time
```
